Assets/Scripts/EnemyController.cs

```diff
@@ -2,9 +2,11 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+public enum MovementType { PATH, MONGO };
```

Comment:
This is an enum, and is very useful for representing lists of types. See below for usage, but doing this makes sure you don't have to type "path" as a string over and over, but more importantly makes sure that you don't accidentally typo something and try to check against "Path" and end up causing a bug that's hard to track down.